packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 #### salesforcedx-vscode-core
 
-- Use regex to match sub-directories in when clause ([PR #4224](https://github.com/forcedotcom/salesforcedx-vscode/pull/4224))
+- Fixed an issue where you couldn't access the `SFDX Create Apex Class` command from a folder under the `classes` directory in a project. The command is now available by right clicking on any folder in the classes directory ([PR #4224](https://github.com/forcedotcom/salesforcedx-vscode/pull/4224))
```

Comment:
```suggestion
- Fixed an issue where you couldn't access the `SFDX Create Apex Class` command from a folder under the `classes` folder in a project. The command is now available by right clicking on any folder in the `classes` folder. ([PR #4224](https://github.com/forcedotcom/salesforcedx-vscode/pull/4224))
```

Reply:
I realize that that's a lot of "folders".  But I think it's best to stick with one term (i.e. folder instead of directory, because they both mean the same thing).  

Alternatively, you could shorten the whole thing to something like "You can now access the `SFDX Create Apex Class` command when you right-click on a sub-folder of the `classes` project folder."